docs/installation.rst

```diff
@@ -12,7 +12,8 @@ Installing with pip
 Stable version
 """"""""""""""
 
-Modin can be installed with pip both on Linux and Windows. To install the most recent stable release run the following:
+Modin can be installed with pip both on Linux and Windows. Also MacOS is supported for Dask_ engine (see below). 
```

Comment:
This is still wrong :) there is Ray for MacOS on `pypi`, so Modin with Ray can be installed on MacOS via `pip` but cannot via `conda`.

Reply:
Crap! Fixed